### ISSUE TITLE
Export web fonts

### ISSUE THREAD

**Issue by aminb** [CatharsisFonts/Cormorant#22]
I'd be thankful if you could address my requests:

- update the exported woff and woff2 font files to match latest development, and
- include the web fonts in release bundles.

I also noticed that the OpenType files and UFO files are lacking behind the Glyphs files. Is that intentional? (I think you use Glyphs, but it'd be nice to have UFO files up to date with so that I could open them in FontForge)

Thanks for the great work!

**Comment by CatharsisFonts** [comment]
Dear Amin,

yeah, I'm sticking to TrueType during development. For now, I've updated the OT, UFO, and webfont files in the commit. I'll add a dedicated webfont ZIP-file during the next Release, which will most likely happen once the Cyrillic review is finished.

Cheers!

**Comment by aminb** [comment]
Hi Christian,

Thank you so much!

I understand updating all the formats during development is somewhat a hassle, so it'd be great to have them updated on each release instead.

Thanks again,
Amin

P.S. looking forward to the next release!
